src/main/java/tutorial/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+spring.data.mongodb.uri=mongodb+srv://admin:admin@yummy.ocgk0lm.mongodb.net/?retryWrites=true&w=majority
```

Comment:
Don't push this file to GitHub since it contains the database key. The `application.properties` file must be made at `src/main/java/resources/` not `src/main/java/tutorial/resources`. The extra directory is why the .gitignore failed to stop the file from pushing. Can you move the resources directory to the java directory and delete the tutorial directory? That should fix the issue